README.md

```diff
@@ -41,6 +42,12 @@ This requires a free FMOD account.
     - `api/studio/lib/x64/fmodstudio.dll`
     - `api/studio/lib/x64/fmodstudio_vc.lib`: **rename to `fmodstudio.lib`**
 
+### Linux
+- Download the "FMOD Engine" package for Linux. Make sure to select [a compatible version](#versioning).
+- Create a new folder `fmod` in the root of your project.
+- Extract the `api` folder into it.
```

Comment:
Can't we replicate the behavior of windows of putting these files into the root?

Reply:
IMO that will get real ugly real soon, as that's 12 files you would be adding to your project root. And if you are shipping the game for different target architectures on linux you need the folders anyways. And if we add the Steam Audio plugin later it's going to be even more lib files in the root.

Reply:
It's also less work for the user this way

Reply:
In that case, we should streamline this for all platforms, **including** Windows. Not sure what convention to use. I've seen in other projects written in different languages the use of a `./vendor` folder.

Reply:
If we use a different folder than root for Windows then Windows users will also need to use a build.rs... Maybe let's decide on this in a different PR/issue?